README.md

```diff
@@ -824,6 +824,7 @@ Table of Contents
    * [shrinkray.io](https://shrinkray.io/) — Free image optimization of GitHub repos
    * [tinypng.com](https://tinypng.com/) — API to compress and resize PNG and JPEG images, offers 500 compressions for free each month
    * [transloadit.com](https://transloadit.com/) — Handles file uploads and encoding of video, audio, images, documents. Free for Open source, charities, and students via the GitHub Student Developer Pack. Commercial applications get 2 GB free for test driving
+   * [uploadcare.com](https://uploadcare.com/) — End-to-end file platform, fast and secure. It covers a complete media pipeline: uploading, processing, and delivery
```

Comment:
please add a bit of details about the features available for free

Reply:
Almost all the features are available for developers for free.
Features like: Smart Crop, Image and Video Transformations, Adaptive Delivery, RESTful file management and Webhooks, File Uploading API and UI, Image CDN and Origin Services, Smart Compression and many others

Reply:
```Yes, there are free Uploadcare offerings for Developers, Startups and Non-profits. We will need more information about your project to enable the Developer tier, while Startup and Non-profit offerings require signing the respective contract. Every kind of a free Uploadcare tier is time-limited and is tailored to the specific needs of an interested party.```

what kind of time-limit would you impose typically?

Reply:
> `Yes, there are free Uploadcare offerings for Developers, Startups and Non-profits. We will need more information about your project to enable the Developer tier, while Startup and Non-profit offerings require signing the respective contract. Every kind of a free Uploadcare tier is time-limited and is tailored to the specific needs of an interested party.`
> 
> what kind of time-limit would you impose typically?

You are absolutely right!
The typical time-limit is 3 months but everything is discussable so it can be 4 months or so.

Reply:
That wouldnt be suitable to the list then I am afraid

Reply:
> That wouldnt be suitable to the list then I am afraid

Even though it is completely free and may speed up finishing the project?
Btw, anyone can re-register as a new member using a bit different data, hehe